chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -77,30 +86,31 @@ class labels as values, which is obtained from inference.
         the number of images in the input.
```

Comment:
`a list of (H, W) or (N, H, W)` looks good to me. In this case, the arguments should be `pred_labels` and `gt_labels`.